jdl/validators/business-error-checker.js

```diff
@@ -140,7 +140,10 @@ class BusinessErrorChecker {
             return;
         }
         const skippedUserManagement =
-            this.applicationSettings.skippedUserManagement || this.jdlObject.getOptionsForName(OptionNames.SKIP_USER_MANAGEMENT)[0];
+            this.applicationSettings.skippedUserManagement ||
+            this.jdlObject
+                .getApplications()
+                .some(jdlApplication => jdlApplication.getConfigurationOptionValue(OptionNames.SKIP_USER_MANAGEMENT));
```

Comment:
The validation is tedious here.
We may be able do something easier here I think: remove any validation related to user/authority management and do it once as a special case later, instead of injecting the boolean in each validation part.
What do you think?